lib/utils/note-utils.test.js

```diff
@@ -55,13 +50,13 @@ describe('noteTitleAndPreview', () => {
 
   it('should have enough text for an Expanded preview, even if the title is very long', () => {
     // Longer than the char limits
-    const title = 'foo'.repeat(titleCharacterLimit);
-    const paragraph = 'foo'.repeat(previewCharacterLimit);
+    const title = 'foo'.repeat(64);
+    const paragraph = 'foo'.repeat(200);
 
     note.data.content = title + '\n' + paragraph;
     const result = noteTitleAndPreview(note);
-    expect(result.title).toHaveLength(titleCharacterLimit);
-    expect(result.preview).toHaveLength(previewCharacterLimit);
+    expect(result.title).toHaveLength(64);
+    expect(result.preview).toHaveLength(200);
```

Comment:
if we export these values in `note-utils` then we can import them here and then our test won't break when we make an intentional change to the note length and also we will couple the test more tightly with the code it's testing.

Reply:
I don't like that we export something just to use it in a test.

Reply:
I made the change but I don't like it.

Reply:
> I don't like that we export something just to use it in a test.

why's that?